test/Microsoft.ML.Tests/Scenarios/Api/Metacomponents.cs

```diff
@@ -41,5 +41,103 @@ public void Metacomponents()
                 var predictor = trainer.Train(new TrainContext(trainRoles));
             }
         }
+
+        /// <summary>
+        /// OVA with calibrator argument
+        /// </summary>
+        [Fact]
+        public void New_OVAWithExplicitCalibrator()
+        {
+            var dataPath = GetDataPath(IrisDataPath);
+
+            using (var env = new TlcEnvironment())
+            {
+                var calibrator = new PavCalibratorTrainer(env);
+
+                var data = new TextLoader(env, MakeIrisTextLoaderArgs())
+                    .Read(new MultiFileSource(dataPath));
+
+                var sdcaTrainer = new LinearClassificationTrainer(env, new LinearClassificationTrainer.Arguments { MaxIterations = 100, Shuffle = true, NumThreads = 1 }, "Features", "Label");
+                var pipeline = new MyConcatTransform(env, "Features", "SepalLength", "SepalWidth", "PetalLength", "PetalWidth")
```

Comment:
>pipeline [](start = 20, length = 8)

call `TestEstimatorCore(pipeline, data)`

Reply:
this will run a bunch of tests

---
In reply to: [217526588](https://github.com/dotnet/machinelearning/pull/865#discussion_r217526588) [](ancestors = 217526588)

Reply:
couldn't use it because of the MyConcatTransform in the pipeline. It's GetSchemaOuput throws. Will update once the concat estimator is in.

---
In reply to: [217526641](https://github.com/dotnet/machinelearning/pull/865#discussion_r217526641) [](ancestors = 217526641,217526588)

Reply:
Well, maybe take it out of the pipeline? :)

---
In reply to: [217551711](https://github.com/dotnet/machinelearning/pull/865#discussion_r217551711) [](ancestors = 217551711,217526641,217526588)